Makefile

```diff
@@ -31,15 +31,15 @@ build:
 	cargo build --all-targets $(PROFILE_FLAG) $(VERBOSE_RUN)
 
 test: # runs unit tests
-	cargo test --workspace --exclude integration --exclude integration-dockerized $(PROFILE_FLAG) $(VERBOSE_RUN) -- $(NOCAPTURE)
+	cargo test --workspace --exclude integration --exclude epoch_coordinator $(PROFILE_FLAG) $(VERBOSE_RUN) -- $(NOCAPTURE)
```

Comment:
Is there a way to tag tests? To not list them one by one here?

Reply:
let's skip it until we have more packages. Maybe there is a nice way of covering it, but I haven't discovered it yet